authentication/jsonld-contexts/authentication.jsonld

```diff
@@ -0,0 +1,11 @@
+{
+	"@context": {
+		"name": "https://ontology.eglobalmark.com/authentication#name",
+		"username": "https://ontology.eglobalmark.com/authentication#username",
+		"role": "https://ontology.eglobalmark.com/authentication#role",
```

Comment:
or `roles` as it is not meant to be unique?

Reply:
```
"roles": {
  "type": "Property",
  "value": ["admin","creator"]
}
```

is this how you see it?